pkg/agent/flowexporter/connections/conntrack_connections.go

```diff
@@ -227,7 +227,7 @@ func (cs *ConntrackConnectionStore) AddOrUpdateConn(conn *flowexporter.Connectio
 		klog.V(4).InfoS("Antrea flow updated", "connection", existingConn)
 	} else {
 		cs.fillPodInfo(conn)
-		if conn.Mark == openflow.ServiceCTMark.GetValue() {
+		if conn.Mark == openflow.ServiceCTMark.GetValueWithOffset() {
```

Comment:
I think the point of using bits is to support the case that a connection can be set with multiple bits, which means we shouldn't match the entire value, otherwise there wouldn't be any difference from the previous code.
It should use `con.Mark & openflow.ServiceCTMark.Mask == openflow.ServiceCTMark.Value`

Reply:
You are right, `conn.Mark` may be a union value of more than one marks.